helm-chart/templates/nextcloud-aio-apache-deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    kompose.cmd: kompose convert -c -f latest.yml
+    kompose.version: 1.27.0 (b0ed6a2c9)
+  creationTimestamp: null
```

Comment:
```suggestion
```

This is a leftover I presume. It's probably a good idea to remove `creationTimestamp` from all manifests.

Reply:
done